donkeycarManagerFront/src/components/dashboard.vue

```diff
@@ -173,7 +173,8 @@
             <vs-td>
               <vs-row>
                 <vs-col vs-type="flex" vs-justify="center" vs-align="center" w="6">
-                  <vs-button warn v-if="jobs.state !== 'PAUSING' && jobs.state !== 'PAUSED' && jobs.state !== 'RESUMING' && jobs.state !== 'CANCELLING'" @click="pauseJobs(jobs)" > Pause </vs-button>
+                  <vs-button loading v-if="reload===true"> reload</vs-button>
+                  <vs-button v-if="jobs.state !== 'PAUSING' && jobs.state !== 'PAUSED' && jobs.state !== 'RESUMING' && jobs.state !== 'CANCELLING' && reload === false" @click="reload(jobs)" > reload </vs-button>
```

Comment:
It's called `jobs` with and `s` at the end and it's only `one job` (not a list of job**s**), you should rename it it's confusing.